src/Core/Framework/Adapter/Cache/CacheIdLoader.php

```diff
@@ -9,6 +9,9 @@
 use Shopware\Core\Framework\Uuid\Uuid;
 use Symfony\Component\Messenger\EventListener\StopWorkerOnRestartSignalListener;
 
+/**
+ * @deprecated tag:v6.7.0 - Will be removed without replacement and is not needed anymore - reason:remove-subscriber
```

Comment:
please properly deprecate and throw deprecation notices in the methods, so users are notified that this will be removed

for the one place where we still call it when the major flag is not active we should use Feature::callSilent()

Reply:
that deprecation will trigger always, so it's useless thats why i didnt done that

Reply:
that's the only way we can make sure that it is not used in some other weird places

Reply:
and how do you want to ignore it that it's not called for any request?

Reply:
@keulinho done

Reply:
reverted intentionally?

Reply:
yes feature checks soo early are not working